.env.example

```diff
@@ -31,9 +31,11 @@ REGISTER_TEMPLATE = mb-votingapp-signup
 REGISTER_EMAIL_TAG = votingapp_signup
 REGISTER_MB_SOURCE = votingapp
 
-MAILCHIMP_GROUP_ID = 55555
-MAILCHIMP_GROUP_NAME = VotingApp
 MC_OPT_IN_PATH = 555555
+MAILCHIMP_LIST_US = 555555
+MAILCHIMP_LIST_BR = 555555
+MAILCHIMP_LIST_MX = 555555
+MAILCHIMP_LIST_GLOBAL = 555555
```

Comment:
Yep, already did! Both Cats Gone Good and Celebs Gone Good (prod) have the appropriate environment variables for testing this once it gets merged and deployed.